### PR TITLE
Clean up model input signature handling for tuples

### PR DESCRIPTION
**Pull request by karlhigley**
Partial fix for https://github.com/NVIDIA-Merlin/Merlin/issues/361#issuecomment-1191889455

### CODE REVIEW

File: merlin/systems/dag/ops/operator.py [hunk 1]
```diff
@@ -297,3 +298,27 @@ def _schema_to_dict(schema: Schema) -> dict:
         }
 
     return schema_dict
+
+
+def _add_model_param(params, paramclass, col_schema, dims=None):
+    dims = dims if dims is not None else [-1, 1]
+    if col_schema.is_list and col_schema.is_ragged:
```
Comment (nv-alaiacano) [inline merlin/systems/dag/ops/operator.py:305]: Could we update `test_tf_op_exports_own_config` to give it a model with a tupled input as well so this new logic gets covered?

**Review by nv-alaiacano — APPROVED** [review]
lgtm aside from the one useful regression test.

I assume this `_add_model_param` function is living in `operator.py` instead of `tensorflow.py` because we expect to use it for other model frameworks as well (besides FIL), is that right?

File: merlin/systems/dag/ops/tensorflow.py [hunk 2]
```diff
@@ -143,38 +116,69 @@ def _export_model(self, model, name, output_path, version=1):
...
         )
 
+        config.parameters["TF_GRAPH_TAG"].string_value = "serve"
+        config.parameters["TF_SIGNATURE_DEF"].string_value = "serving_default"
+
+        for _, col_schema in self.input_schema.column_schemas.items():
+            _add_model_param(config.input, model_config.ModelInput, col_schema)
+
+        for _, col_schema in self.output_schema.column_schemas.items():
+            _add_model_param(config.output, model_config.ModelOutput, col_schema)
```
Comment (nv-alaiacano) [inline merlin/systems/dag/ops/tensorflow.py:130]: This logic would likely also need to be used in `PipelineableInferenceOperator.export` https://github.com/NVIDIA-Merlin/systems/blob/572dbf154015924106718d21f1896037d8a3bab6/merlin/systems/dag/ops/operator.py#L259-L274

File: merlin/systems/dag/ops/operator.py (hunk 1 above)
Comment (karlhigley) [reply merlin/systems/dag/ops/operator.py:305]: Added a new test to cover the schema inference from the model file

**Pull request merged**